src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/xlf/LocalizableStrings.de.xlf

```diff
@@ -1,21 +1,21 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
   <file datatype="xml" source-language="en" target-language="de" original="../LocalizableStrings.resx">
     <body>
       <trans-unit id="Authoring_InvalidJsonElementInLocalizationFile">
         <source>Localization file should only contain elements with type 'string'. Remove elements that are not strings.</source>
-        <target state="new">Localization file should only contain elements with type 'string'. Remove elements that are not strings.</target>
+        <target state="translated">Die Datei Lokalisierungsdatei darf nur Elemente mit dem Typ „Zeichenfolge“ enthalten. Entfernen Sie die Elemente, die keine Zeichenfolgen sind.</target>
         <note />
       </trans-unit>
       <trans-unit id="Authoring_InvalidManualInstructionLocalizationIndex">
         <source>In localization file under the post action with id '{1}', there are localized strings for manual instruction(s) with ids '{0}'. These manual instructions do not exist in the template.json file and should be removed from localization file.</source>
-        <target state="new">In localization file under the post action with id '{1}', there are localized strings for manual instruction(s) with ids '{0}'. These manual instructions do not exist in the template.json file and should be removed from localization file.</target>
+        <target state="translated">In der Lokalisierungsdatei unter der POST-Aktion mit der ID „{1}“ befinden sich lokalisierte Zeichenfolgen für manuelle Anweisungen mit den IDs „{0}“. Diese manuellen Anweisungen sind in der Datei „template.json“ nicht vorhanden und sollten aus der Lokalisierungsdatei entfernt werden.</target>
```

Comment:
@cristianosuzuki77 note that we have one test which uses German, and it cannot properly display/decode these quotes. I'm not sure if it's due to tests are running on US locale, however want to ask if it's a common practice to use such quotes for CLI output? 

I'm fixing the test anyway to ignore quotes.

Reply:
I will try to setup de-DE VM tomorrow to see if it works.